boards/arm/nrf52840dk_nrf52840/nrf52840dk_nrf52840.dts

```diff
@@ -120,6 +120,7 @@
 		sw1 = &button1;
 		sw2 = &button2;
 		sw3 = &button3;
+		bootloader-led0 = &led0;
```

Comment:
@anangl Is that OK to add such aliases for all the Nordic boards like that?

Reply:
Does it need to be a separate alias, can't just `led0` be used?

Reply:
MCUboot already accepted such alias for led it uses. It not necessary must be referenced to the board's led0.
Is this additional alias a problem?

Reply:
I see. I should have looked at the mcuboot changes before asking...
I don't see such alias as a problem, just wondered if it should be added in Zephyr when only mcuboot uses it. But I guess it should be okay, as the alias is generic enough to be used also by other bootloaders if needed.